lua/goto-preview.lua

```diff
@@ -30,6 +30,7 @@ local M = {
     force_close = true, -- passed into vim.api.nvim_win_close's second argument. See :h nvim_win_close
     bufhidden = "wipe", -- the bufhidden option to set on the floating window. See :h bufhidden
     stack_floating_preview_windows = true, -- Whether to nest floating windows
+    open_new_window_for_current_file = true, -- Wheter to open a new floating window for a reference within the current file
```

Comment:
💅 
```suggestion
    same_file_float_preview = true, -- Whether to open a new floating window for a reference within the current file
```

Just a naming suggestion here and a typo fix. I'm not sure the suggestion is a better name but thought I'd at least put it here for discussion